omod/src/main/webapp/template/headerFull.jsp

```diff
@@ -178,7 +176,7 @@
 				<div id="openmrs_msg"><openmrs:message code="${msg}" text="${msg}" arguments="${msgArgs}"  htmlEscape="false" /></div>
 			</c:if>
 			<c:if test="${err != null}">
-				<div id="openmrs_error"><openmrs:message code="${err}" text="${err}" arguments="${errArgs}" htmlEscape="false"/></div>
+				<div id="openmrs_error"><openmrs:message code="${err}" text="${err}" arguments="${errArgs}" htmlEscape="true"/></div>
```

Comment:
@dkayiwa do you know why the htmlEscaping here was set to "false" originally? Can you think of anything in the legacy UI that would break by turning it on?